stack.yaml

```diff
@@ -3,4 +3,4 @@ resolver: lts-13.2
 packages:
```

Comment:
In order to make it build with `stack` on CI you also need to update `resolver` here. Just replace `lts-13.2` with `nightly-2019-01-15`